app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -96,38 +65,9 @@ def available_providers
       current_provider_user.providers
     end
 
-    def set_sub_navigation_items
+    def set_workflow_flags
```

Comment:
Good name

Reply:
@duncanjbrown my bad, renamed the view file so I need to amend here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3448/files#diff-bb6c9c9a4b198f788c032f0c8e5ef49818058b557fdd36003af44dc577ca906aR29